src/monali/Assignment_8.java

```diff
@@ -0,0 +1,78 @@
+package monali;
+
+public class Assignment_8 {
+	
+	
+	
+	void evenNumber(int startnum , int endnum) {
+		 while(startnum <= endnum) {
+	    	 if(startnum %2 == 0)
+			System.out.println("even numbers are" +startnum);
+			startnum = startnum + 1;
+			}
+	}
+	     
+	 void printNumber(int startnum1 , int endnum1) {
+		 while(startnum1 <= endnum1) {
+			if(startnum1 %5 == 0)
+			System.out.println("number divisible by 5 is" +startnum1);
+			startnum1 = startnum1 + 1;
+		}
+		
+	 }
+
+	 void printNum(int totalnum) {
+		 int startnum2 = 0;
```

Comment:
You don’t require to create extra variable, you can handle this without **startnum2**, think on this
We require method which accepts two arguments (start and end)

please refer below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/219/files

Reply:
above both methods as the requirement but below twos are not